tools/clean/atlas/network_peering.go

```diff
@@ -8,16 +8,28 @@ import (
 	"go.mongodb.org/atlas-sdk/v20231001002/admin"
 )
 
-func (c *Cleaner) listNetworkPeering(ctx context.Context, projectID string) []admin.BaseNetworkPeeringConnectionSettings {
-	peers, _, err := c.client.NetworkPeeringApi.
-		ListPeeringConnections(ctx, projectID).
-		Execute()
-	if err != nil {
-		fmt.Println(text.FgRed.Sprintf("\tFailed to list networking peering for project %s: %s", projectID, err))
+var (
+	SupportedProviders = []string{"AWS", "AZURE", "GCP"}
```

Comment:
In `atlas.go` there are already constants with the supported cloud providers, please reuse them

Reply:
ok